plugins/API/tests/System/expected/test_AutoSuggestAPITest_eventValue__VisitsSummary.get_range.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <result>
-	<nb_visits>0</nb_visits>
-	<nb_actions>0</nb_actions>
-	<nb_visits_converted>0</nb_visits_converted>
+	<nb_visits>4</nb_visits>
```

Comment:
Not sure why this test didn't used to work. Ran the test with float and double and in both cases the value looked like they were stored correctly. 
This is what the fixture data looks like
![image](https://user-images.githubusercontent.com/273120/72578501-1ec06380-393b-11ea-88ae-d32295659b9a.png)

Reply:
It didn't work as the comparison `custom_float = '345.678'` doesn't return any results when the field is a float. The number seems not to be stored accurate enough to be able to compare it correctly. When changing the column to `DOUBLE` it gets converted to `345.6780090332031`.

Not sure if it would make sense to truncate some minor fraction digits after changing the type 🤔 

Using a `DOUBLE` the number is stored accurate enough to be compared correctly

Reply:
Could use something like `DOUBLE(15,5)` or so and limit it to 5 numbers after the  dot. I just tested it and the problem is that then a similar issue as with decimal happens that all numbers are formatted to have 5 digits after dot
![image](https://user-images.githubusercontent.com/273120/74378607-20eae480-4e4b-11ea-8d30-32486eb9ef34.png)

Reply:
Could we maybe additionally run a script like `UPDATE log_link_visit_action SET custom_float = TRUNCATE(custom_float, 5)` after it was converted to double? That should throw away all fraction digits after the fifth one 🤔

Reply:
@sgiehl from what I see mysql stores also new values like this. 

and truncate seems to ensure there's always that many zeros too?
![image](https://user-images.githubusercontent.com/273120/74474464-70441a00-4f0a-11ea-8a8a-25b170d636f8.png)

Reply:
If we are using `DOUBLE` only (without defining any fixed fraction digits), that should work, doesn't it?

Reply:
Yes, that's why I ended up only defining double

Reply:
Guess that's fine, but it might be useful to truncate the fraction digits on update, so the comparison of values works better after the update 🤔

Reply:
@sgiehl not sure I understand this fully? If we just use `DOUBLE` then this shouldn't be needed?

Reply:
When we currently have a value like `345.678` and the field is converted to `DOUBLE`, the new value will be something like `345.6780090332031`. That's the reason why I suggested to run something like a TRUNCATE, so we have a similar value as before

Reply:
Which MySQL version is that? This wasn't happening for me on MySQL 8. Or maybe I'm not checking correctly?

Reply:
mysql 5.7

Reply:
I see, can reproduce it on `10.2-MariaDB`. I just created 3 rows with values 3, 4.1 and 4.12. After converting to float I got this:

![image](https://user-images.githubusercontent.com/273120/74888165-13e96a80-53e2-11ea-934a-369c2ceaf00c.png)

After executing your suggested query I got this:

![image](https://user-images.githubusercontent.com/273120/74888205-31b6cf80-53e2-11ea-9b46-9841e44ad431.png)

So it's still changing the values quite a bit. 

It seems there might not be a safe way to change the field? I wonder if it's worth it or maybe we rather create an FAQ for people needing more accuracy and only try to change it for new installs.

@mattab there doesn't seem to be an easy way to change the field for this column. I reckon we could really rather only change it for new installs maybe and create an FAQ for all other cases.